engine/runtime-with-polyglot/src/test/java/org/enso/interpreter/test/instrument/VerifyJavaScriptIsAvailableTest.java

```diff
@@ -41,11 +41,13 @@ public static void initEnsoContext() {
   public static void closeEnsoContext() throws Exception {
     ctx.close();
 
+    /* XXX: Is this needed for startup?
```

Comment:
Why were we even doing that in `AfterClass`? This should probably go completely

Reply:
Because this used to check that during the existence of `ctx`, the right message is called.

Reply:
Updated in ef1c79825a